apps/customer-portal/backend-v2/.env.example

```
@@ -0,0 +1,17 @@
+# Entity service (cs-tools/entity-service)
+ENTITY_SERVICE_BASE_URL=http://localhost:8081
+# Optional — only needed if entity-service sits behind a gateway that requires
+# OAuth2 client-credentials auth. Leave unset for a local direct-to-service run.
+OAUTH2_CLIENT_ID=
+OAUTH2_CLIENT_SECRET=
+OAUTH2_TOKEN_URL=
+ENTITY_SERVICE_SCOPES=
+
+# Auth — set to false for local testing (skips JWT signature verification)
+AUTH_JWKS_ENDPOINT=
+AUTH_ISSUER=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AUTH_ISSUER key should go before the AUTH_JWKS_ENDPOINT key

(UnorderedKey)